doc/api/tls.markdown

```diff
@@ -694,6 +694,11 @@ This is an encrypted stream.
 A proxy to the underlying socket's bytesWritten accessor, this will return
 the total bytes written to the socket, *including the TLS overhead*.
 
+## Class: CleartextStream
+
+The CleartextStream class in Node.js version v0.10.39 and prior has been
```

Comment:
@jasnell Do we mean in all v0.10.x versions? Node v0.10.40 was just released, and this seems a bit confusing as it would mean that `CleartextStream` is available in that version.